service/src/main/java/uk/gov/hmcts/reform/fpl/service/CommonCaseDataExtractionService.java

```diff
@@ -1,18 +1,50 @@
 package uk.gov.hmcts.reform.fpl.service;
 
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.config.HmctsCourtLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.model.Applicant;
+import uk.gov.hmcts.reform.fpl.model.ApplicantParty;
+import uk.gov.hmcts.reform.fpl.model.Child;
+import uk.gov.hmcts.reform.fpl.model.ChildParty;
 import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+import uk.gov.hmcts.reform.fpl.model.HearingVenue;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.common.JudgeAndLegalAdvisor;
+import uk.gov.hmcts.reform.fpl.model.docmosis.DocmosisChild;
+import uk.gov.hmcts.reform.fpl.model.docmosis.DocmosisHearingBooking;
+import uk.gov.hmcts.reform.fpl.model.docmosis.DocmosisJudgeAndLegalAdvisor;
+import uk.gov.hmcts.reform.fpl.model.docmosis.DocmosisRespondent;
 
 import java.time.LocalDateTime;
 import java.time.format.FormatStyle;
+import java.util.List;
 import java.util.Optional;
 
+import static java.time.format.FormatStyle.LONG;
+import static java.util.Optional.ofNullable;
+import static java.util.stream.Collectors.toList;
+import static org.apache.commons.lang3.ObjectUtils.defaultIfNull;
 import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.DATE_TIME;
 import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.formatLocalDateTimeBaseUsingFormat;
 import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.formatLocalDateToString;
+import static uk.gov.hmcts.reform.fpl.utils.JudgeAndLegalAdvisorHelper.formatJudgeTitleAndName;
+import static uk.gov.hmcts.reform.fpl.utils.JudgeAndLegalAdvisorHelper.getLegalAdvisorName;
 
 @Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class CommonCaseDataExtractionService {
+    private static final String DEFAULT = "BLANK - please complete";
+
+    private final HmctsCourtLookupConfiguration hmctsCourtLookupConfiguration;
+    private final HearingVenueLookUpService hearingVenueLookUpService;
+
+    String getCourtName(String localAuthority) {
```

Comment:
Do we want the methods in here to be package private? I know that with our package structure they can be but should they be public?

Reply:
public would provide the same access to this method as package private, so I don't see why it needs to be declared

Reply:
That is what I was thinking, was just wondering whether it was intentional or not